tools/invoke-tests

```diff
@@ -43,7 +43,7 @@ db="$build_directory/cover_db"
 export PERL5LIB="$source_directory:$source_directory/ppmclibs/blib/lib:$source_directory/ppmclibs/blib/arch/auto/tinycv:$PERL5LIB"
 if [[ $WITH_COVER_OPTIONS ]]; then
     path_regex="^|$source_directory/|\\.\\./"
-    select="($path_regex)(OpenQA|backend|consoles|ppmclibs)/|($path_regex)isotovideo|($path_regex)[^/]+\.pm,-ignore,external/|\.t|data/tests/|fake/tests/|/CheckGitStatus.pm|$prove_path"
```

Comment:
It doesn't look like coverage tracking for `data/tests` actually works. The report shows only percentages greater than zero because lines that would be evaluated at compile time are considered after all.

Reply:
Sure. I don't really care about data/tests/, only t/. We don't need to ignore data/tests/ or fake/tests/ though when it does not harm